tests/test_prism.py

```
@@ -774,10 +775,26 @@ def test_end_to_end_prismify_merge_artifact_merge(xlsx, template):
 
     # so we can merge
     original_ct[PROTOCOL_ID_FIELD_NAME] = prism_patch[PROTOCOL_ID_FIELD_NAME]
-    
+
+    if template.type in SUPPORTED_ANALYSES:
+        if template.type == 'cytof_analysis':
+            # creating `cytof_input_patch` which is different from `prism_patch`
+            cytof_input_xlsx_path = os.path.join(TEMPLATE_EXAMPLES_DIR,'cytof_template.xlsx')
+            cytof_input_xlsx, _ = XlTemplateReader.from_excel(cytof_input_xlsx_path)
+            cytof_input_template = Template.from_type('cytof')
+            cytof_input_patch, _, _ = prismify(cytof_input_xlsx, cytof_input_template)
+
+            # cytof_template.xlsx and cytof_analysis_template.xlsx need not have the same Protocol ID
+            cytof_input_patch[PROTOCOL_ID_FIELD_NAME] = original_ct[PROTOCOL_ID_FIELD_NAME]
+
+            original_ct, errs = merge_clinical_trial_metadata(cytof_input_patch, original_ct)
+
     # "prismify" provides only a patch so we need to merge it into a "full" ct
     full_after_prism, errs = merge_clinical_trial_metadata(prism_patch, original_ct)
-    assert not errs
+
+    # CyTOF analysis patch being invalid on it's own
+    if not template.type == 'cytof_analysis':
+        assert not errs
```

So it ignores errors for `cytom_analysis`?